pkg/metrics/prometheus.go

```diff
@@ -479,7 +479,7 @@ func (h *histogram) Observe(value float64) {
 	labels := h.labelNamesValues.ToLabels()
 	collector := h.hv.With(labels)
 	collector.Observe(value)
-	h.collectors <- newCollector(h.name, labels, collector, func() {
+	h.collectors <- newCollector(h.name, labels, h, func() {
```

Comment:
@marco-jantke  could you review this. It compiles but I'm not sure it does what it's supposed to.

Reply:
Sorry for the late response, I'm trying to check that out tomorrow. In the meantime did you verify whether histogram values are still exported properly via the `/metrics` endpoint?

Reply:
to be honnest I just made sure it compiles, I did not test it.

Reply:
currently the metrics have not been implemented in Traefik v2.
This PR can not be tested and therefore it would be better if this PR is closed until we have restored the functionality.